content/blog/activation-memory-a-deep-dive-using-pytorch.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
@kevinmusgravehpe 
Hi Kevin, the external link is missing for the newly created blog link card.
Could you please add it to your blog link card entry?

Thanks
Denis